modules/distribution/src/repository/resources/conf/templates/repository/conf/identity/identity.xml.j2

```diff
@@ -346,39 +411,42 @@
             provide 'A:foo.com', foo.com should be the user store domain name and 'A'
             represent the relavant mapping of token store table, i.e. tokens will be
             added to a table called IDN_OAUTH2_ACCESS_TOKEN_A. -->
-        <AccessTokenPartitioningDomains>{{oauth.token.partitioning.domains}}</AccessTokenPartitioningDomains>
+        <AccessTokenPartitioningDomains><!-- A:foo.com, B:bar.com --></AccessTokenPartitioningDomains>
         <AuthorizationContextTokenGeneration>
-            <Enabled>{{oauth.token.validation.context.enable}}</Enabled>
-            <TokenGeneratorImplClass>{{oauth.token.validation.context.token_generator}}</TokenGeneratorImplClass>
-            <ClaimsRetrieverImplClass>{{oauth.token.validation.context.claim_retriever}}</ClaimsRetrieverImplClass>
-            <ConsumerDialectURI>{{oauth.token.validation.context.consumer_dialect}}</ConsumerDialectURI>
-            <SignatureAlgorithm>{{oauth.token.validation.context.signature_alg}}</SignatureAlgorithm>
-            <AuthorizationContextTTL>{{oauth.token.validation.context.ttl}}</AuthorizationContextTTL>
+            <Enabled>{{oauth.token.validation.include_validation_context_as_jwt_in_reponse}}</Enabled>
+            <TokenGeneratorImplClass>{{oauth.extensions.token_context_generator}}</TokenGeneratorImplClass>
+            <ClaimsRetrieverImplClass>{{oauth.extensions.token_context_claim_retriever}}</ClaimsRetrieverImplClass>
+            <ConsumerDialectURI>{{oauth.extensions.token_context_dialect_uri}}</ConsumerDialectURI>
+            <SignatureAlgorithm>{{oauth.token_validation.validation_response_signing_algorithm}}</SignatureAlgorithm>
+            <AuthorizationContextTTL>{{oauth.token.validation.validation_response_jwt_validity}}</AuthorizationContextTTL>
         </AuthorizationContextTokenGeneration>
 
         <!-- Configurations for JWT bearer grant. Supported versions: IS 5.8.0 onwards. -->
         <JWTGrant>
             <!-- Validate issued at time (iat) of JWT token. The validity can be set using 'IATValidity' configuration.
-            Default value is 'true'.
-            -->
-            <EnableIATValidation>{{oauth.jwt.iat_validation}}</EnableIATValidation>
-
+             Default value is 'true'.
+             -->
+            <EnableIATValidation>true</EnableIATValidation>
             <!-- Reject the JWT if the iat of JWT is pass a certain time period. Time period is in minutes.
-            'EnableIATValidation' configuration should be set to 'true' in order to make use of the validity period.
-            Default value is '30' minutes.
+             'EnableIATValidation' configuration should be set to 'true' in order to make use of the validity period.
+             Default value is '30' minutes.
              -->
-            <IATValidityPeriod>{{oauth.jwt.iat_validity_period}}</IATValidityPeriod>
+            <IATValidityPeriod>30</IATValidityPeriod>
```

Comment:
Is this something user should not change?

Reply:
This is a new configuration. Need to template after discussing with the team.